static/bots/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Guide to rate limiting and bot development
+---
+
+When creating a bot, scraper or automation to programmatically get data from the ONS site and APIs, please read the following guidance.
+
+## Rate limiting
+
+The ONS dissemination sites and API apply rate limiting to ensure a high-quality service is delivered to all users.
+
+The following rate limits have been applied:
+
+* 120 requests per 10 seconds - all site and API assets
+
+* 200 requests per 1 minute - all site and API assets
+
+* 15 requests per 10 seconds - high demand site assets
+
+The time for which you will be blocked varies and is dynamically changed by the ONS, depending on release schedules and observed demand.
+
+If you exceed these limits, the API will return a `429 Too Many Requests` HTTP status code and a `Retry-After` header containing the number of seconds until you may try your request again.
+
+The ONS expects that you programmatically respect the Retry-After header and back-off your polling accordingly. If this is not respected, our algorithms will automatically impose a total block to our services for anything up to 24 hours.
+
+Your tooling should respect the `Retry-After` header and pause your requests for the specified duration. If this is not respected our algorithms may impose a block to our services for up to 1 hour.
+
+Read more in our [fair usage policy](TODO)
```

Comment:
We should probably include something to say that we reserve the right to change these at any time. Perhaps this paragraph is the correct place?

```suggestion
We reserve the right to change our rate limits at any time and without prior notice. Read more in our [fair usage policy](TODO)
```

Reply:
Also this URL will be based on the page title (which is actually going to be `fair use policy`) so will be generated as:

```suggestion
Read more in our [fair use policy](https://www.ons.gov.uk/help/fairusepolicy).
```
Also added missing full stop.